src/authenticators/direct_authenticator/mod.rs

```diff
@@ -49,9 +55,10 @@ mod test {
 
         let app_name = "app_name".to_string();
         let req_auth = RequestAuth::new(app_name.clone().into_bytes());
+        let conn_metadata = ConnectionMetadata::NoMetadata;
```

Comment:
You'll need to remove these and just pass `None` 😄

Reply:
Doh -- fixed. :)